CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Unreleased
 
+- #930, Split table comment on newline to get OpenAPI operation summary and description - @daurnimator
```

Comment:
This should go under "Added" or "Fixed" depending on how you classify the change.